packages/mds-compliance-batch-processor/batch_process.ts

```diff
@@ -45,7 +45,7 @@ export async function computeSnapshot() {
   })
   const compliance_as_of = now() // The timestamp right after we fetch the inputs (latest state as of that time)
 
-  const provider_ids = Object.keys(await getProviders())
+  const provider_ids = (await ProviderServiceClient.getProviders()).map(p => p.provider_id)
```

Comment:
This appears often enough that I wonder if it warrants its own service method. NBD.